.cmdb/test_product_id/config/solutionsv2/alm/environment.json

```diff
@@ -0,0 +1,7 @@
+{
+  "Environment" : {
+    "Id" : "alm",
+    "Name" : "alm",
+    "MultiAZ" : False
```

Comment:
strange value for Json file (@roleyfoley probably)

Reply:
Ye, this is just a dummy cmdb which is not supposed to work in real. In the future, we should replace it with something functional.

Reply:
I thought you are going to write unit-tests which are using this demo cmdb. It won't be useful for real users anyway, they will have to either delete it or create another with different name. We never know what will they use, alm or not alm, ecs or not ecs and so on.

Reply:
I suppose that testing will be the story of its own. I mean, currently, there is nothing to test there except options propagation. Maybe in the future, we'll be able to do tests of local files content but still, it will be somewhat hard to test actual results of script execution which involves changing the real infrastructure.

Reply:
> strange value for Json file (@roleyfoley probably)

@koriaf Thanks for catching that. I've fixed it up in the cookie cutter pattern, so that it's not an issue for new deployments

Reply:
@Neketek so please fix it in this file as well